### DISCUSSION TITLE
`console_focus_follow` only when latest flow is focused

### DISCUSSION THREAD

**Discussion by italankin** [mitmproxy/mitmproxy#5596] [Ideas]
Add an option (new mode or something) for `console_focus_follow` such that focus will follow new flows only when latest flow is focused.

The behavior is the following:
* When the latest flow is selected, every new flow will gain cursor focus (the same way `console_focus_follow` works now)
* If the user focuses another flow (the oldest, for example), new flows will not steal focus from selected

It is useful when you receive many requests, but want to inspect one particular flow, without applying a filter or turning `console_focus_follow` off.

**Comment by mhils** [comment]
That sounds super reasonable to me (mitmweb is doing something like that already). I'd be happy to merge a PR that adds suchs an "auto" mode and make it the default 😃
